### DISCUSSION TITLE
Slice sampler

### DISCUSSION THREAD

**Discussion by dirmeier** [blackjax-devs/blackjax#510] [New features]
Hello all, I was wondering if you plan on implementing a conventional (ideally multivariate) slice sampler at some point? I know TFP has an implementation, but it would be nice to have it available here, too.  I have a (badly written but working) JAX implementation which could be used as a starting point. Thanks!

Cheers, Simon

**Comment by albcab** [comment]
Hi! There is no plan of implementing a multivariate slice sampler. If you want to implement it for multivariate distributions (the hit-and-run approach like TFP, I'm guessing) you are welcome to. Or you can open an issue for it, possibly share your current implementation, and someone might pick it up.
